### ISSUE TITLE
4D groupwise registration not supported in currrent ITKElastix

### ISSUE THREAD

**Issue by ViktorvdValk** [InsightSoftwareConsortium/ITKElastix#70]
ITKElastix should be rebuild with support for 4D images to use the 4D groupwise image registration of Elastix

**Comment by thewtex** [comment]
As a first step, we want to update the wrapping configuration:

https://github.com/InsightSoftwareConsortium/ITKElastix/blob/6a4dd1d3ccf58303d5747508bf2be3785f014be6/wrapping/itkElastixRegistrationMethod.wrap

From using `itk_wrap_image_filter` to `itk_wrap_template`. Example:

https://github.com/InsightSoftwareConsortium/ITK/blob/master/Modules/Filtering/ImageStatistics/wrapping/itkLabelStatisticsImageFilter.wrap

But we want to change 

```
foreach(d ${ITK_WRAP_IMAGE_DIMS})
```

to

```
UNIQUE(dims "${ITK_WRAP_IMAGE_DIMS};4")
foreach(d ${dims})
```

along with the wrapped pixel types.

**Comment by thewtex** [comment]
Addressed with update to ITK 5.2 RC 2.
